build.gradle

```
@@ -195,16 +205,15 @@ tasks.withType(JavaCompile).configureEach {
 }
 
 // Register different Skript testing tasks
-createTestTask('quickTest', 'src/test/skript/environments/' + latestEnv, false, latestJava, false)
-createTestTask('skriptTestJava17', 'src/test/skript/environments/java17', false, latestJava, false)
-createTestTask('skriptTestJava8', 'src/test/skript/environments/java8', false, oldestJava, false)
-createTestTask('skriptTestDev', 'src/test/skript/environments/' + (project.property('testEnv') == null
-	? latestEnv : project.property('testEnv') + '.json'), true, Integer.parseInt(project.property('testEnvJavaVersion') == null
-	? latestJava : project.property('testEnvJavaVersion')), false)
+String env = project.property('testEnv') == null ? latestEnv : project.property('testEnv') + '.json'
+int envJava = project.property('testEnvJavaVersion') == null ? latestJava : Integer.parseInt(project.property('testEnvJavaVersion') as String)
```

```suggestion
int envJava = project.property('testEnvJavaVersion') == null ? latestJava : Integer.parseInt(project.property('testEnvJavaVersion'))
```
It'll already be a string

---

`project.property` returns Object; and yes it works without explicit cast but then IntellIJ screams at you.